examples/cpp/route_guide/Makefile

```diff
@@ -86,7 +86,7 @@ ifneq ($(HAS_VALID_PROTOC),true)
 	@echo "Please install Google protocol buffers 3.0.0 and its compiler."
 	@echo "You can find it here:"
 	@echo
-	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-alpha-1"
+	@echo "   https://github.com/google/protobuf/releases/tag/v3.0.0-beta-2"
```

Comment:
Since this file has been updated, please update it's copyright to 2015-2016, otherwise the sanity test will fail:
`examples/cpp/route_guide/Makefile modified 2016 but copyright only extends to 2015`

Same for helloworld.